weekly-updates/project-lead-meetings/08_05_2018.md

```diff
@@ -136,15 +136,28 @@ _Proposed Deadline_:
 ## snackru
 
 _UPDATES_:
+LCS login works and it checks LCS for mentor role
 
 _QUESTIONS_:
+can you make amulya an org member
+LCS /auth takes 6 secs on my machine?!?!?!
```

Comment:
bcrypt is slow, srry.

Reply:
From what I've read it seems like it is slow, but not that slow. Something I read said people were experiencing slowdowns from .1ms to 100ms not 6000ms? I think meteor uses bcrypt for it's default login and it seemed at least under 500ms

Reply:
after checking on my machine bycript.checkpw doesn't take 6 seconds, but my machine is not an aws machine. aparently you can tune bcrypt to optimally take around 240ms by adjusting the number of rounds when you make the salt. you can do bcrypt.generate(10) to get a smaller amount of rounds (the default seems to be 12)